sdk/src/wallet/migration/chrysalis.rs

```diff
@@ -310,6 +191,7 @@ fn decrypt_record(record_bytes: Vec<u8>, encryption_key: &[u8; 32]) -> crate::wa
     String::from_utf8(pt).map_err(|e| Error::Migration(format!("{:?}", e)))
 }
 
+#[allow(unused)]
```

Comment:
Is it to convoluted to figure out under what combination(s) they are used ?

Reply:
No, the problem is the combinations are not useful. it would be rocksdb or (storage and stronghold). just makes more sense to do this

Reply:
Fair enough